modules/gdnative/gdnative_api.json

```diff
@@ -2933,6 +2933,17 @@
           ["const godot_pool_byte_array *", "p_pba"]
         ]
       },
+      {
+        "name": "godot_array_slice",
```

Comment:
This needs to be part of the Core 1.1 extension and not placed in here, as this will cause binary compatibility issues.

Reply:
In my latest push I've incremented the core api version.

Reply:
That's not what I meant :smile: I meant that this entry for `godot_array_slice` needs to live inside `core/next/next/api` (The Core 1.2 extension) instead of the `core/api` where it is currently located.

Reply:
Ahhh, ok :laughing: sorry my bad. I've moved the slice json object into the core/next/next/api section in my latest force push and gotten rid of the change in the python script.